src/components/Breadcrumb/index.vue

```diff
@@ -22,7 +22,7 @@
 </template>
 
 <script lang="ts">
-import pathToRegexp from 'path-to-regexp'
```

Comment:
Why you change this and add the "any" below? I didn't see any lint errors from my side. If you have met some errors, can you take a screenshot for that?

Reply:
Here is, when I click dashboard, `pathToRegexp.(path)` will throw `TypeError: Cannot read property 'compile' of undefined` because of undefined `pathToRegexp`. So I change
```ts
import pathToRegexp from 'path-to-regexp'
const toPath = pathToRegexp.compile(path)
```
 to
```ts
import { compile } from 'path-to-regexp'
const toPath = compile(path)
```

![image](https://user-images.githubusercontent.com/31433480/74600209-f1e1a680-50c8-11ea-9eb7-45f7229a3002.png)

![image](https://user-images.githubusercontent.com/31433480/74600208-eb532f00-50c8-11ea-9eeb-33b5cded5a6a.png)

And the lint error is 

![image](https://user-images.githubusercontent.com/31433480/74600585-75ea5d00-50ce-11ea-8a11-446367ce003d.png)

Reply:
具体原因可以参考下这个issue下的评论 https://github.com/Armour/vue-typescript-admin-template/issues/127#issuecomment-600429047

Reply:
Thanks!